controllers/register.controller.js

```diff
@@ -0,0 +1,30 @@
+const Tutor = require('../models/tutor.model');
+const Student = require('../models/student.model');
+const jwt = require('jsonwebtoken');
+
+const createUser = async (req, res) => {
+  try {
+    const { type, inputs } = req.body;
+
+    let userSchema = '';
+
+    if (type === 'student') {
+      userSchema = Student;
+    } else {
+      userSchema = Tutor;
+    }
```

Comment:
Excellent implementation, you can refactor a little more but It's not necessary. 

`userSchema = (type === 'student') ? Student : Tutor`

Reply:
Applied your advice, it looks more elegant with the ternary operator.